lib/resty/signal.lua

```diff
@@ -124,14 +122,15 @@ function _M.kill(pid, sig)
 
         signum = tonumber(resty_signal.resty_signal_signum(id))
         if signum < 0 then
-            error("missing C def for signal ", sig)
+            error("missing C def for signal: ", sig)
```

Comment:
Do no agree with this. "missing C def for signal FOO" reads better.

Reply:
Ok, yeah I can see that too. Will revert

Reply:
Addressed.